src/components/image/index.tsx

```diff
@@ -95,13 +98,14 @@ export type ImageProps = RNImageProps &
      * The image height
      */
     height?: string | number;
+    source: ImageSourceType;
```

Comment:
What about other props, like `errorSource`, that use the type `ImagePropTypes.source`?

Reply:
It could have been nice to get rid of the `ImagePropType` altogether...

Reply:
You should change the types in the props declaration as well...